srtcore/srt_compat.c

```diff
@@ -26,7 +26,7 @@ written by
 #include <features.h>
 #endif
 
-#if defined(_WIN32) || defined(WIN32)
+#if defined(_WIN32) || defined(_WIN32)
```

Comment:
Here you caused a tautology. :)

Reply:
Yeah, problems from using sed instead of looking for each occurrence manually.